lib/Constants.js

```diff
@@ -181,7 +181,8 @@ module.exports.MessageTypes = {
     GUILD_DISCOVERY_REQUALIFIED: 15,
     GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING: 16,
     GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING: 17,
-    REPLY: 19
+    REPLY: 19,
+    APPLICATION_COMMAND: 20
```

Comment:
Can you add this in the relevant TS doc as well

Reply:
Shall do that now